src/targets/gpu/gemm_impl.cpp

```diff
@@ -140,7 +140,7 @@ void gemm_impl(context& ctx,
             compute_type = rocblas_datatype_f32_r;
     }
 
-#if ROCBLAS_VERSION_MAJOR >= 2 && ROCBLAS_VERSION_MINOR >= 38
+#if ROCBLAS_VERSION_MAJOR >= 3 || ROCBLAS_VERSION_MAJOR == 2 && ROCBLAS_VERSION_MINOR >= 38
```

Comment:
These checks can be completely removed. This is a really old version of rocblas.

Reply:
yes, older then rocm 4.3

Reply:
removed